.circleci/config.yml

```diff
@@ -282,6 +287,7 @@ jobs:
       - slack-notify-job-failure
       - deploy-lock-remove-on-fail
 
+  # Build server binaries and run tests (but not blazor, which is separated out for build speed)
   build-fsharp-backend:
     executor: in-container
     resource_class: xlarge
```

Comment:
can/should this be lowered? (Not sure if xlarge costs us more per CI run. maybe large is enough now since blazor compile has been extracted)

Reply:
It does cost us 2x. 
I thought about this. However, builds still aren't exactly fast, so I decided not to turn it down. It does seem like there's a lot of CPU being used, and I was recently looking into whether I could use even larger machines.

I'm continuing the optimization of builds in a new branch shortly. If I see the CPU drop I'll switch to smaller machine.